docs/source/tagset/elemento-abstract.rst

```diff
@@ -3,25 +3,20 @@
 <abstract>
 ----------
 
-Aparece em
+Aparece em:
   :ref:`elemento-article-meta`
  
-Ocorre
+Ocorre:
   Zero ou mais vezes
 
 
-Tag que identifica o resumo do artigo e não deve conter informação de 
-atributo ``@xml:lang``. Embora em via de regra esse elemento ocorra 
-zero ou mais vezes, ele se faz obrigatório quando :ref:`elemento-article` for declarado
-com o atributo ``@article-type="research-article"`` ou ``@article-type="review-article"``.
-Em ``<abstract>`` deve ser inserido um elemento ``title`` para especificar o título do resumo.
+Elemento que identifica o resumo de um artigo. Não deve conter o atributo @xml:lang. Embora <abstract> possa não ocorrer, faz-se obrigatório quando <article-meta> for declarado com atributo @article-type="research-article" ou @article-type="review-article".
```

Comment:
Oi Asael, na realidade eu me perdi nas trocas de email. Ainda hoje vou fazer um levantamento das pendências que tenho contigo.

-Gustavo

On 27 de junho de 2016 at 12:11:26, Asael C. Silva (notifications@github.com) wrote:
> > Zero ou mais vezes
> >
> >
> > -Tag que identifica o resumo do artigo e não deve conter informação de
> > -atributo ``@xml:lang``. Embora em via de regra esse elemento ocorra
> > -zero ou mais vezes, ele se faz obrigatório quando :ref:`elemento-article` for declarado  
> > -com o atributo ``@article-type="research-article"`` ou ``@article-type="review-article"``.  
> > -Em ```` deve ser inserido um elemento ``title`` para especificar o título  
> do resumo.
> > +Elemento que identifica o resumo de um artigo. Não deve conter o atributo @xml:lang.  
> Embora possa não ocorrer, faz-se obrigatório quando for  
> declarado com atributo @article-type="research-article" ou @article-type="review-article".  
>  
> Gustavo e meu post? Parece que vocês ignoraram minha dúvida.
>  
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/scieloorg/scielo_publishing_schema/pull/245/files/361efbbbe84028fe1ab0e4f46675de2fd9b50cfa#r68593960